inventory/Inventory.cs

```diff
@@ -88,6 +88,21 @@ private InventorySlot Slot(int slot)
         return GetNode<InventorySlot>($"MarginContainer/InventoryGrid/InventorySlot{slot + 1}");
     }
 
+    public void DeductHealth(int amount)
+    {
+        int numDeducted = 0;

```

Comment:
```suggestion
        var numDeducted = 0;

```
local variables can be typed lazily with `var`